code/controllers/subsystem/job.dm

```diff
@@ -1,3 +1,9 @@
+/// Defines for locations of items being added to your inventory on spawn
+#define LOCATION_LPOCKET "in your left pocket"
+#define LOCATION_RPOCKET "in your right pocket"
+#define LOCATION_BACKPACK "in your backpack"
+#define LOCATION_HANDS "in your hands"
```

Comment:
Probably don't need defines for this as they're only used once, you can just put the strings in the list

Reply:
It's undefined at the end.
Defines are only used by compiler, there is no reason to create list, you are going to use once and not modify.